packages/flutter/lib/src/foundation/tree_diagnostics_mixin.dart

```diff
@@ -628,15 +660,23 @@ abstract class DiagnosticsNode {
 
   /// Whether the diagnostics should be hidden when showing the default
   /// view of a tree.
+  ///
+  /// This could be set to true (hiding the property) if another property
+  /// provides a normally more useful summary of the value.
```

Comment:
Also, the more typical case is hiding a property where the properties value is uninteresting. For example, matching the typical default value for the property.